Gems/AWSCore/Code/Tools/ResourceMappingTool/utils/environment_utils.py

```diff
@@ -38,6 +39,19 @@ def setup_qt_environment(bin_path: str) -> None:
     new_path = os.pathsep.join([binaries_path, path])
     os.environ['PATH'] = new_path
 
+    # On Linux, we need to load pyside2 and related modules as well
+    if platform.system() == 'Linux':
+        import ctypes
+
+    preload_shared_libs = [f'{bin_path}/libpyside2.abi3.so.5.14',
```

Comment:
question: will this break if we update the pyside2 version? If so is there a test we could add which would catch this?

Reply:
Yes it will. This is not a long term solution for this problem, because this similar behavior is being done in the QtForPython and Python Binding gems since we use our own frozen version of python.

Reply:
Looks like the indentation is off :-(

Reply:
could we have a jira to track this issue? what is the long term solution?

Reply:
The long term solution will require an not to patch pyside2 and import it as a module, but to use python's native 3.7 pyside2 module instead so that it will be imported as part of o3de's installed package during the pip install requirements